nginx/replace-api-url.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
```

Comment:
This is debatable, but i would avoid this shebang. It makes it look that this is compatible with POSIX `sh`.
But in reality (ln 10), we have `[[ ]]` that is undefined: https://mywiki.wooledge.org/BashFAQ/031

That's why it is safer to have the following `#!/usr/bin/env bash`

Reply:
We have Alpine Linux as base docker images - there is no `bash` there, so I've used `sh` instead.

Reply:
Okay makes sense. Then one should do `if [ -z "${API_URL}" ]; then` instead of using `[[ ]]`